programs/account-compression/tests/merkle_tree_tests.rs

```diff
@@ -699,9 +701,33 @@ async fn functional_1_initialize_state_merkle_tree_and_nullifier_queue(
         merkle_tree_pubkey,
     )
     .await;
+
+    assert_eq!(merkle_tree.deserialized().index, 1);
+    // TODO(vadorovsky): Assert fees.
```

Comment:
when do you plan to add these asserts?

Reply:
fyi no need to assert the tip we will refactor that to the compressed pda program and just keep the rollover fees in account compression